server.js

```diff
@@ -1,33 +1,142 @@
+If statement for if the length of the specificBook variable/array is 0 (i.e. the author doesn't exist) then return the error message, else translate the bookID element from the array into json so the data it's readable on the endpoint.*/
+//http://localhost:8081/books/10
+app.get('/books/:id', (request, response) => {
+  const bookID  = request.params.id;
+  const specificBook = booksData.find(item => item.bookID === +bookID);
+  
+  if (!specificBook) {
+    response.status(404).json("Sorry that ID doesn't correspond to any book in our API");    
+  }   
+  response.json(specificBook);    
+});
+
+// Created an object outlining the documentation. Can be found on the start page of the api url
+const documentation = {
+  "Endpoint 1": {
+    "https://books-deployment.herokuapp.com/books": "Returns the entire books array",
+
+    "https://books-deployment.herokuapp.com/books?author=choose": "Returns books by a specific author using the author query string parameter. It will return an array with one or more elements, if the author isn't valid then you will get an error message.",
+
+    "https://books-deployment.herokuapp.com/books?title=choose": "Use this endpoint to return books with a specific title using the title query string parameter. It will return an array with one or more elements, if the author isn't valid then you'll get an error message.",
+
+    "https://books-deployment.herokuapp.com/books?language=choose":  "Use this endpoint to return books written in a specific language using the title query string parameter. It will return an array with one or more elements. The following languages are valid: eng, en-GB, en-US, spa, fre, ger, ara, por, grc, mul. If the language isn't valid then you'll get an error message.",
+    
+    "https://books-deployment.herokuapp.com/books?averagerating=choose": "Use this endpoint to return books with a specific average rating using the averagerating query string parameter. It will return an array with one or more elements. If the average rating isn't found you'll get an error message.",
+
+    "https://books-deployment.herokuapp.com/books?page=choose": "This endpoint uses the page query parameter and allows you to query up to 10 pages which will return 50 books per page. You can use this to implement pages in the frontend.",
+  },
+  
+  "Endpoint 2": {
+    "https://books-deployment.herokuapp.com/books/:id": " Use this endpoint to return books with a specific id and replace :id with a number.",
+  },
+};
+
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+  /* --- Third endpoint --
+Path for my api documentation to be found which is the homepage of the url
+*/
+app.get('/documentation', (request, response) => {
+  response.json(documentation);
+});
```

Comment:
The documentation is awesome, [here](https://www.npmjs.com/package/express-list-endpoints)'s a cool lib that you can try out next time.